# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/linux-firmware/hotfix-ln-sf.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- a/copy-firmware.sh.vanilla	2025-12-16 19:17:32.512560453 +0100
+++ b/copy-firmware.sh	2025-12-16 19:18:04.098227802 +0100
@@ -149,16 +149,16 @@
     if test -e "$target"; then
         $verbose "creating link $l -> $t"
         if [ "$num_jobs" -gt 1 ]; then
-            echo "ln -s \"$t\" \"$destdir/$l\"" >> "$parallel_args_file"
+            echo "ln -sf \"$t\" \"$destdir/$l\"" >> "$parallel_args_file"
         else
-            ln -s "$t" "$destdir/$l"
+            ln -sf "$t" "$destdir/$l"
         fi
     else
         $verbose "creating link $l$compext -> $t$compext"
         if [ "$num_jobs" -gt 1 ]; then
-            echo "ln -s \"$t$compext\" \"$destdir/$l$compext\"" >> "$parallel_args_file"
+            echo "ln -sf \"$t$compext\" \"$destdir/$l$compext\"" >> "$parallel_args_file"
         else
-            ln -s "$t$compext" "$destdir/$l$compext"
+            ln -sf "$t$compext" "$destdir/$l$compext"
         fi
     fi
 done
